cache/src/main/java/net/runelite/cache/region/Position.java

```diff
@@ -51,6 +51,18 @@ public static Position fromPacked(int packedPosition)
 		return new Position(x, y, z);
 	}
 
+	// Unpack a position from an int in a client script
+	public static Position fromScriptInt(int scriptInt)
```

Comment:
why is this different from `fromPacked`?

Reply:
Turns out it is the same. Got rid of it